state/upgrades_test.go

```diff
@@ -2108,3 +2110,39 @@ func (s *upgradesSuite) TestAddRelationStatus(c *gc.C) {
 		expectUpgradedData{statuses, expectedStatuses},
 	)
 }
+
+func (s *upgradesSuite) TestCopyMongoSpaceToHASpaceConfigWhenValid(c *gc.C) {
```

Comment:
I am starting to believe that you have a knack for writing exceptionally clear tests.